### PR TITLE
Fix/ios safari video issues

### PR DESCRIPTION
**Pull request by tsubik**
Fixing issues with video player mostly for iOS

- the player will no longer restart itself after the end, it should just close itself,
- fixing fullscreen iPad z-index issues with feedback button and navbar,
- fixing player not showing up on iPhone at all,
- improving interaction with movie progress slider on iOS using https://rangetouch.com/. Looks like only sliding worked and clicking/touching the track didn't.

### CODE REVIEW

File: frontend/scripts/index.ejs [hunk 1]
```diff
@@ -9,4 +9,6 @@
 <div id="app-root-container"></div>
 </body>
 <script type="text/javascript" src="//cdn.transifex.com/live.js"></script>
+<!-- Rangetouch to fix <input type="range"> on touch devices (see https://rangetouch.com) -->
+<script src="https://cdn.rangetouch.com/1.0.5/rangetouch.js"></script>
```
Comment (tiagojsag) [inline frontend/scripts/index.ejs]: cant we use https://github.com/sampotts/rangetouch#node--npm instead?

Comment (tsubik) [reply frontend/scripts/index.ejs]: Done.

**Comment by tsubik** [comment]
@tiagojsag I think that we should allow users to pause video without closing it. To close video the user will have to leave the fullscreen mode or it will close automatically when it ends. Any thoughts?

**Comment by tsubik** [comment]
Ready :rocket:

**Comment by tiagojsag** [comment]
Let's wait for rodrigo to review this

**Comment by tsubik** [comment]
Done :)

**Pull request merged**